common/flatpak-dir.c

```diff
@@ -10520,10 +10673,11 @@ flatpak_dir_uninstall (FlatpakDir                 *self,
 
   if (flatpak_decomposed_is_runtime (ref) && !force_remove)
     {
+      g_autoptr(GHashTable) runtime_app_map = NULL;
```

Comment:
This seems...entirely unused, left behind by accident or similar?

Reply:
It's there because `flatpak_dir_list_app_refs_with_runtime` will assert if you pass it a `NULL` `runtime_app_map` pointer. And I tried rewriting that function to allow `NULL` but it makes the logic quite a bit more complicated, so perhaps not worth it.

Reply:
Fine with me. We can make the logic more complicated when it turns out to be a problem.